tools/jdbc/src/main/java/org/duckdb/DuckDBConnection.java

```diff
@@ -25,14 +25,17 @@
 public class DuckDBConnection implements java.sql.Connection {
 	protected ByteBuffer conn_ref = null;
 	protected DuckDBDatabase db;
-
+    protected String catalog;
+    protected String schema;
 	public DuckDBConnection(DuckDBDatabase db) throws SQLException {
 		if (db.db_ref == null) {
 			throw new SQLException("Database was shutdown");
 		}
 		conn_ref = DuckDBNative.duckdb_jdbc_connect(db.db_ref);
 		DuckDBNative.duckdb_jdbc_set_auto_commit(conn_ref, true);
 		this.db = db;
+		this.catalog = "catalog";
```

Comment:
Why are those constants "catalog"? I think they should not be initialised like so.